chapters/ch10/4.train-and-evaluate-the-model.ipynb

```diff
@@ -110,13 +110,64 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 19,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "[-0.4319807665098383, -0.444478207387438, -0.4675688993325839]"
+       "{'size': 100,\n",
```

Comment:
This looks like OpenSearch format... notebook needs to be re-generated using Solr as the default.

Reply:
Fixed